src/test/shared/sam/debugger/samDebugConfigProvider.test.ts

```diff
@@ -1199,12 +1197,12 @@ Outputs:
                 request: 'launch',
                 debugPort: undefined,
                 port: -1,
-                outFilePath: '',
                 handlerName: 'app.lambda_handler',
                 baseBuildDir: actualNoDebug.baseBuildDir,
                 envFile: `${actualNoDebug.baseBuildDir}/env-vars.json`,
                 eventPayloadFile: `${actualNoDebug.baseBuildDir}/event.json`,
             }
+            delete expectedNoDebug.debugArgs
```

Comment:
if possible, it is better to do these workaround in `assertEqualLaunchConfigs()` instead of per-test.